cmd/relay/bgs/validator.go

```diff
@@ -409,7 +409,7 @@ func (val *Validator) HandleSync(ctx context.Context, host *models.PDS, msg *atp
 	commit, err := atrepo.LoadCARCommit(ctx, bytes.NewReader([]byte(msg.Blocks)))
 	if err != nil {
 		commitVerifyErrors.WithLabelValues(hostname, "car").Inc()
-		if !val.Sync11ErrorsAreWarnings {
+		if !val.Sync11ErrorsAreWarnings || commit == nil {
```

Comment:
I would expect that if `err != nil` that the `commit` is going to be `nil`.  Is that not the case?  I think that means we always have to error if we fail to read the commit from the CAR, there's no case where we'll end-up warning here.

Reply:
ok, yeah, content and err are mutually exclusive, cleaned up